OWML.ModLoader/ModDependency.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OWML.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.ModLoader
+{
+    public class ModDependency
```

Comment:
I think this is deleted in master because it's not in use - you probably don't need it either? :)

Reply:
done